src/utils/number.spec.ts

```diff
@@ -17,3 +17,28 @@ describe("isValidNumber", () => {
     expect(isValidNumber("123123.234234")).toBe(true);
   });
 });
+
+describe("parseNumberString", () => {
+  it("returns null for string values that can't compute to a number", () => {
+    expect(parseNumberString("undefined")).toBe(null);
+    expect(parseNumberString()).toBe(null);
+    expect(parseNumberString("")).toBe(null);
+    expect(parseNumberString(undefined)).toBe(null);
+    expect(parseNumberString(null)).toBe(null);
+    expect(parseNumberString("null")).toBe(null);
+    expect(parseNumberString("not a number")).toBe(null);
+    expect(parseNumberString("kjas;lkjwo;aiej(*&,asd;flkj-")).toBe(null);
```

Comment:
If this and the next assertion are testing for specific characters, can you either extract a variable that describes what the string is testing or update the string itself with this info if it's simpler.

This also applies to the test below.

Reply:
done.